src/app/LoginController.java

```diff
@@ -0,0 +1,22 @@
+package app;
+
+import java.net.URL;
+import java.util.ResourceBundle;
+import javafx.fxml.Initializable;
+
+/**
+ * FXML Controller class
+ *
+ * @author walte
+ */
+public class LoginController implements Initializable {
+
+    /**
+     * Initializes the controller class.
+     */
+    @Override
+    public void initialize(URL url, ResourceBundle rb) {
+        // TODO
```

Comment:
We'll need some test data here, basically creating one account, write it to a file, read it back from the file, and test a password. Doing stuff like this doesn't seem to always work in the initialize method for some reason but you can create and call a private method that does it. The test from TestHarness will just make sure that the password test returns true or false as expected.